cy_canvas/reducers/cyElements.js

```diff
@@ -220,6 +235,17 @@ const cyElements = (state, action) => {
         action: action.action
       }))
 
+    case "DELETE_ELEMENTS":
+      const remain = state.filter(t => 
+        !action.elements.map(getId).includes(t.data.id)
```

Comment:
You're right, I didn't realize that. As you said it might not be a problem but I'll edit that line to avoid it anyway.